### ISSUE TITLE
Object overlay on the chat

### ISSUE THREAD

**Issue by sbadux** [deltachat/deltachat-desktop#129]
Hello,
DeltaChat shows the mailing list emails in the app and I've foud this strange overlay. It seems to happen only with the mailing list.

![image](https://user-images.githubusercontent.com/28478027/46568927-6f31ce80-c94d-11e8-81c9-e5e211c9f512.png)

**Comment by ralphtheninja** [comment]
Duplicate of https://github.com/deltachat/deltachat-desktop/issues/124

**Comment by ralphtheninja** [comment]
Actually, lets re-open it.

What should be the expected behavior?

**Comment by sbadux** [comment]
Sorry @ralphtheninja, this question is specifically addressed to me or it's a generic question?

**Comment by ralphtheninja** [comment]
Generic question :)

**Comment by sbadux** [comment]
Using the last version (30) this happens also in the chat list.

![image](https://user-images.githubusercontent.com/28478027/46916660-d7bc2380-cfbd-11e8-9896-42e6be60daaf.png)

**Comment by ralphtheninja** [comment]
@sbadux yes that's duplicate of https://github.com/deltachat/deltachat-desktop/issues/124

**Comment by ralphtheninja** [comment]
@sbadux So, do you have any input on what the expected behavior should be?

**Comment by sbadux** [comment]
@ralphtheninja I'm really sorry for the mess I'm doing here :-/ I'm a bit confused about bugs, duplicates etc. It's the first project I'm trying to help and I made some mistakes. Sorry.

Anyway, I can suggest some options like:
1 - Move the "Create Chat" and "Create Group" buttons in the hamburger menu
2 - Move the "Create Chat" and "Create Group" at the bottom of the app.
3 - Move the "Create Chat" and "Create Group" at the bottom of the app & move the hamburger menu where the "Logout" button currently is & add the Logout as an option of the menu. 
4 - Add a simple "+" menu where you can choose to create a new group or chat like in the Android app.
5 - Remove the hamburger menu (it only contains the key transfer option)  and add a right colum with these options (Create Chat, Create Group, Key Transfer) if you don't think to add many other options.

I would suggest for the 3rd or 4th option if I can choose.

**Comment by ralphtheninja** [comment]
I think I personally prefer 1. Simply move them into the hamburger menu. It would solve the problem and it would be a minimal change.

**Comment by ralphtheninja** [comment]
To solve the first issue (that you reported), I suggest getting rid of the text that is drawn right next to the hamburger menu, since it's the same information as the information to the top left.

**Comment by ralphtheninja** [comment]
Just to summarize a bit.

This issue is code wise _not_ a duplicate, because it's another view than the chat view. But the behavior is similar. So I'll address this issue and #124 separately.
